File: 1000-1200 Python/TABLET.py
T = int(input())

for i in range(T):
    temp = input().split(" ")
    N, B = int(temp[0]), int(temp[1])
    area = 0

    for j in range(N):
        temp = input().split(" ")
        W, H, P = int(temp[0]), int(temp[1]), int(temp[2])

        if P <= B and W*H > area:
            area = W*H

    if area != 0:
        print(area)
    else:
        print("no tablet")

File: 1000-1200 C/TABLET.c
#include <stdio.h>
int main(){
    int T, N, B, W, H, P;     
    scanf("%d", &T);
    for (int i = 0; i < T; i++){
        scanf("%d %d", &N, &B);
        int area = 0;
        
        for (int j = 0; j < N; j++){
            scanf("%d %d %d", &W, &H, &P);

            if (P <= B && W*H > area){
                area = W*H;
            }
        }

        (area != 0) ? printf("%d\n", area) : printf("no tablet\n"); 
    }   
    return 0;
}
